src/App.js

```diff
@@ -3,12 +3,15 @@ import { Routes, Route } from 'react-router-dom';
 import Categories from './routes/Categories';
 import Books from './routes/Books';
 import Navbar from './components/Navbar';
+import Layout from './components/Layout';
 
 function App() {
   return (
     <>
       <Navbar />
       <Routes>
+
+        <Route path="/" element={<Layout />} />
```

Comment:
- [ ] I noticed that the Navbar is being called twice, once in the Layout component and then again in that App component. Kindly render the Navbar component once so it's not rendered twice on the page. Please take a look at the screenshot below:

<img width="636" alt="Screenshot 2023-03-23 at 12 10 11 AM" src="https://user-images.githubusercontent.com/50941074/227067173-836c3d37-7126-413a-8217-f6ca6cd54906.png">

Reply:
Hello @AmaduKamara  Thank you for noticing the double rendering of the Navbar component. I have already addressed this issue by making the necessary changes to ensure that the Navbar is rendered only once, in the appropriate location. Your feedback is highly appreciated, and please don't hesitate to let me know if you have any further suggestions or notice any other issues.